docs/release-notes.rst

```diff
@@ -0,0 +1,105 @@
+.. _release-notes
+
+Release Notes
+=============
+
+v1.0.0
```

Comment:
Okay I'll do the `0.1.0` version then. I wanted something to signify that we've finished all the work that we originally set out to do, and not just continue with `0.0.6`.